tests/www/test_views.py

```diff
@@ -1395,6 +1395,26 @@ def test_dag_stats_success_for_all_dag_user(self):
         self.check_content_in_response('example_subdag_operator', resp)
         self.check_content_in_response('example_bash_operator', resp)
 
+    def test_dag_stats_success_when_selecting_dags(self):
+        self.logout()
+        self.login(username='all_dag_user',
+                   password='all_dag_user')
```

Comment:
Is this needed? Aren't we already logged in as an admin?

Reply:
Good point, I just checked your right.